guides/doc-Release_Notes/topics/foreman-3.11.4.adoc

```diff
@@ -0,0 +1,16 @@
+= Foreman 3.11.4
```

Comment:
I'm tempted to add a note here for Debian 12 so users who read the release notes just for the version they're upgrading to will see it.
```suggestion
= Foreman 3.11.4

ifdef foreman-deb::[]
Packages for Debian 12 were built with 3.11.0, but our automated pipelines rely on Puppetserver.
Now that Puppetserver packages for Debian 12 are available, this can be considered a supported platform.
endif::[]
```

Reply:
https://github.com/theforeman/foreman-documentation/pull/3393 adds it as a headline feature and with a deprecation. I think having it in both places is a good thing.